templates/ourBlog.hbs

```diff
@@ -0,0 +1,9 @@
+<section class="blog container">
+    
+    <hr>
+    <h2 class="blog__title">{{title}}</h2>
+    <p class="blog__description">{{description}}</p>
+    <a class="blog__button" href="{{assetSrc pdf}}">{{buttonLabel}}</a>
```

Comment:
Please make this link open in a new tab always

Reply:
Done!